xdsl/interpreters/pdl_interp.py

```diff
@@ -131,6 +133,107 @@ def run_getdefiningop(
         )
         return (args[0].owner,)
 
+    @impl_terminator(pdl_interp.CheckOperationNameOp)
+    def run_checkoperationname(
```

Comment:
For the longer names the snake_case is much easier to read IMO
```suggestion
    def run_check_operation_name(
```